doc/reference/reference_lua/fiber.rst

```diff
@@ -779,6 +785,46 @@ recommended.
             - 1448466351270762
             ...
 
+.. _fiber-clock:
+
+.. function:: clock()
+
+    Get the monotonic time in seconds. It is better to use ``fiber.clock()`` for
+    calculating timeouts instead of ``fiber.time()`` because ``fiber.time()``
+    reports real time so it is affected by system time changes.
+
+    :Return: a floating-point number of seconds, representing elapsed wall-clock
+             time since some time in the past that is guaranteed not to change
```

Comment:
since some time in the past = since 01.01.1970? (the Unix epoch is 00:00:00 UTC on 1 January 1970, so most probably here the same starting point is used)

Reply:
No it's not a Unix epoch and "some time in the past" is correct phrase.

It was inspired by [MPI Wtime](https://mpi.deino.net/mpi_functions/MPI_Wtime.html) or [OpeMP get_wtime](https://www.openmp.org/spec-html/5.0/openmpsu160.html)

> a value equal to the elapsed wall clock time in seconds since some time-in-the-past. The actual time-in-the-past is arbitrary, but it is guaranteed not to change during the execution of the application program.

Reply:
Thanks for explaining. Let's leave the text as it is -- it's quite enough for users.